include/gz/sim/rendering/MarkerManager.hh

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2019 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+#ifndef GZ_GAZEBO_MARKERMANAGER_HH_
+#define GZ_GAZEBO_MARKERMANAGER_HH_
+
+#include <memory>
+#include <string>
+
+#include <gz/sim/rendering/Export.hh>
```

Comment:
```suggestion
#include <gz/sim/config.hh>
#include <gz/sim/rendering/Export.hh>
```

to get `IGNITION_GAZEBO_VERSION_NAMESPACE`

Reply:
[5f1458a](https://github.com/gazebosim/gz-sim/pull/1989/commits/5f1458ac6c8c9014b1aa44b9aa44110002ac47d5)